### PR TITLE
EKS POC

### PR DESCRIPTION
**Pull request by yakloinsteak**
## Description

Gets the app closer to being ready for running on EKS without (purposefully) breaking the current ECS deployments. A deployment to ECS that succeeds should mostly vet this code.

## Type of change
- [x] New feature (adds functionality)
- [x] Dependency update

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/jobs/base_job.rb [hunk 1]
```diff
@@ -5,67 +5,48 @@
...
-
-    setup_notifier('DelayedJobRestarter')
-    @notifier.ping(msg) if @send_notifications
-  end
-
-  def expected_path
-    Rails.cache.fetch('deploy-dir') do
-      # A default for the first deployment and local development
-      # This should be set on deployment.
-      Delayed::Worker::Deployment.deployed_to
```
Comment (eanders) [inline app/jobs/base_job.rb:56]: Calls to `deployed_to` or setting of the variable additionally appear in:
```
config/deploy.rb
config/initializers/delayed_job.rb
lib/tasks/cas.rake
```

If we're removing this concept, we should make sure it gets pulled out of those locations as well.

**Review by eanders — APPROVED** [review]
Thanks @yakloinsteak, I think this looks good, we should make sure to deploy to QA before merging.

File: app/jobs/base_job.rb (hunk 1 above)
Comment (yakloinsteak) [reply app/jobs/base_job.rb:56]: Sounds good. I'll do that.

**Comment by yakloinsteak** [comment]
I just deployed this successfully to QA. Did you want to check anything out before merging? Or is there anything else you want me to do to test?

**Comment by eanders** [comment]
Seems to be working, I just confirmed email and I believe DJ.
Let's merge, but I'd like to retarget `release-66`

**Pull request merged**